packages/web-components/src/components/ic-back-to-top/test/basic/__snapshots__/ic-back-to-top.spec.tsx.snap

```diff
@@ -32,7 +32,7 @@ exports[`ic-back-to-top should render with an icon only: should render with an i
 `;
 
 exports[`ic-back-to-top should render with footer: should render with footer 1`] = `
-<ic-footer class="footer footer-light footer-sparse footer-ungrouped light">
+<ic-footer class="footer-light footer-sparse footer-ungrouped ic-footer ic-footer-light">
```

Comment:
should the 'footer-light' class still exist when 'ic-footer-light' has been created?

Reply:
footer-light was to do with the foreground colour, so I've updated it to "ic-footer-foreground-light" now